### ISSUE TITLE
Display chart details on hover

### ISSUE THREAD

**Issue by tschomer** [looker-open-source/malloy#862]
![image](https://user-images.githubusercontent.com/40474956/195203269-392fdde7-a620-4e13-a583-1170c3b70069.png)

There might be an option to turn on some default behavior with a switch in the code. 

I'd love to be able to customize how it looks, if possible.

**Comment by whscullin** [comment]
It looks vega tooltip plugin allows you to style their tooltips in any way you like, but you only have a single div to work with. Not sure if there would be an easy way to do a tail.

**Comment by tschomer** [comment]
i don't think we need the tail on the UI. Is there a way to turn on their default behavior, so we can see how far away we might be from something we want?
